README.md

```diff
@@ -8,65 +8,262 @@ Warns and then closes issues and PRs that have had no activity for a specified a
+| [exempt-pr-labels](#exempt-pr-labels)                               | Labels on PRs exempted from stale                        |                       |
+| [only-labels](#only-labels)                                         | Only issues/PRs with ALL these labels are checked        |                       |
+| [only-issue-labels](#only-issue-labels)                             | Only issues with ALL these labels are checked            |                       |
+| [only-pr-labels](#only-pr-labels)                                   | Only PRs with ALL these labels are checked               |                       |
+| [any-of-labels](#any-of-labels)                                     | Only issues/PRs with ANY of these labels are checked     |                       |
+| [any-of-issue-labels](#any-of-issue-labels)                         | Only issues with ANY of these labels are checked         |                       |
+| [any-of-pr-labels](#any-of-pr-labels)                               | Only PRs with ANY of these labels are checked            |                       |
+| [operations-per-run](#operations-per-run)                           | Max number of operations per run                         | `30`                  |
+| [remove-stale-when-updated](#remove-stale-when-updated)             | Remove stale label from issues/PRs on updates/comments   | `true`                |
+| [remove-issue-stale-when-updated](#remove-issue-stale-when-updated) | Remove stale label from issues on updates/comments       |                       |
+| [remove-pr-stale-when-updated](#remove-pr-stale-when-updated)       | Remove stale label from PRs on updates/comments          |                       |
+| [debug-only](#debug-only)                                           | Dry-run                                                  | `false`               |
+| [ascending](#ascending)                                             | Order to get issues/PRs                                  | `false`               |
+| [skip-stale-issue-message](#skip-stale-issue-message)               | Skip adding stale message on stale issues                | `false`               |
+| [skip-stale-pr-message](#skip-stale-pr-message)                     | Skip adding stale message on stale PRs                   | `false`               |
+| [start-date](#start-date)                                           | Skip stale action for issues/PRs created before it       |                       |
+| [delete-branch](#delete-branch)                                     | Delete branch after closing a stale PR                   | `false`               |
+| [exempt-milestones](#exempt-milestones)                             | Milestones on issues/PRs exempted from stale             |                       |
+| [exempt-issue-milestones](#exempt-issue-milestones)                 | Override [exempt-milestones](#exempt-milestones)         |                       |
```

Comment:
would it make sense for these overrides to be slightly more explicit in the docs like `Override exempt-milestones for issues only`? It might be good to be explicit about what is happening and detail what `override` means in this context

Reply:
I will make the change tonight

Reply:
@luketomlinson done 46352757a0418ff8ddd8cd41d21323ed28add94a